### PR TITLE
Adds toy holoparasites

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

adds toy holoparasites of all types. These can be found in the autodrobe, hacked donksoft vendor, syndicate uplink, and as an arcade prize.  Toy holoparasites are useless and nothing but a liability to the owner.
makes gravity holopara flavor text less lame

## Why It's Good For The Game

a useless new holoparasite for you non antag peasants

## Changelog
:cl:
add: toy holoparasite
tweak: changed gravity holopara flavor text
imageadd: empty clown deck and clown hypo
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
How exactly are they a liability? Are they actually player control, or just wind up toys?

**Comment by zeskorion** [comment]
> How exactly are they a liability? Are they actually player control, or just wind up toys?

they have the downsides of a holopara and none of the upsides. IE dusting on death, damage transfer, making you valid....

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	tech_fluff_string = "<span class='holoparasite'>Boot sequence complete. Standard combat modules locked. Holoparasite swarm online.</span>"
+	carp_fluff_string = "<span class='holoparasite'>CARP CARP CARP! You caught one! It's weak and useless. Can I have a refund?.</span>"
+	var/battlecry = "HONK"
+
+/mob/living/simple_animal/hostile/guardian/toy/verb/Battlecry()
+	set name = "Set Battlecry"
+	set category = "Guardian"
+	set desc = "Choose what you shout as you limply slap people."
+	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input)
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/toy.dm:18]: You need to add a check for the chat filter.
```
if(CHAT_FILTER_CHECK(input))
    [some warning message]
    return
```

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		battlecry = input
+
+
+
+/mob/living/simple_animal/hostile/guardian/toy/AttackingTarget()
+	. = ..()
+	if(isliving(target))
+		say("[battlecry]!!", ignore_spam = TRUE)
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/toy.dm:30]: `ignore_spam = TRUE`

🤔

Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/toy.dm:30]: I am unsure, it's the same battle cry code as that of a standard holoparasite. I believe that it is what makes it not spam several messages in a row and instead put a number behind it

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/toy.dm:18]: sure.

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm (hunk 2 above)
Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/toy.dm:30]: ah, it's so it doesnt IC mute you

**Comment by zeskorion** [comment]
tfw chat filter ruins pr

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/toy.dm:18]: considering the pain the chat filter is putting me through here, i just wont include it so travis passes. i'll paste the code and let you make your code suggestions. this bypass is already in the game so it'll need to get fixed either way, shrug. 

	if(CHAT_FILTER_CHECK(input))
    	to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
    	return

**Comment by zeskorion** [comment]
ugh, travis keeps giving me that error on 21, pushed it and shit, ima go back to bed

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 3]
```diff
@@ -15,9 +15,13 @@
 	set name = "Set Battlecry"
 	set category = "Guardian"
 	set desc = "Choose what you shout as you punch people."
-	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
 	if(input)
-		battlecry = input
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm]:
```suggestion
			to_chat(src, "<span class='warning'>Your battlecry may not include prohibited words! Consider rereading the server rules.</span>")
```

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 4]
```diff
@@ -15,9 +15,13 @@
 	set name = "Set Battlecry"
 	set category = "Guardian"
 	set desc = "Choose what you shout as you punch people."
-	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm:18]: Why are you deleting this line you fool?

Also github doesn't support code suggestions on deleted lines so I can't fix this for you.

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 5]
```diff
@@ -15,9 +15,13 @@
...
 	if(input)
-		battlecry = input
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
+        		return
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm]:
```suggestion
        	return
```

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 6]
```diff
@@ -15,9 +15,13 @@
...
+		else
+			battlecry=input
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm]:
```suggestion
			battlecry = input
```

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+	set name = "Set Battlecry"
+	set category = "Guardian"
+	set desc = "Choose what you shout as you limply slap people."
+	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input)
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
+        	return
+		else
+			battlecry=input
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/toy.dm]:
```suggestion
			battlecry = input
```

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+	var/battlecry = "HONK"
+
+/mob/living/simple_animal/hostile/guardian/toy/verb/Battlecry()
+	set name = "Set Battlecry"
+	set category = "Guardian"
+	set desc = "Choose what you shout as you limply slap people."
+	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input)
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='holoparasite'>Your battlecry may not include forbidden words! Consider rereading the server rules.</span>")
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/toy.dm]:
```suggestion
			to_chat(src, "<span class='warning'>Your battlecry may not include prohibited words! Consider rereading the server rules.</span>")
```

File: code/modules/vending/autodrobe.dm [hunk 9]
```diff
@@ -130,7 +130,8 @@
 		           /obj/item/shield/riot/roman/fake = 1,
 				   /obj/item/clothing/suit/chaplainsuit/clownpriest = 1,
 				   /obj/item/clothing/head/clownmitre = 1,
-		           /obj/item/skub = 1,)
+		           /obj/item/skub = 1,
+				   /obj/item/guardiancreator/toy = 1,)
```
Comment (ike709) [inline code/modules/vending/autodrobe.dm]:
```suggestion
				   /obj/item/guardiancreator/toy = 1)
```

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 10]
```diff
@@ -17,7 +17,12 @@
 	set desc = "Choose what you shout as you punch people."
 	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
 	if(input)
-		battlecry = input
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='warning'>Your battlecry may not include prohibited words! Consider rereading the server rules.</span>")
+        	return
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm]: Github broke the formatting of this line and replaced tabs with spaces.

File: code/modules/mob/living/simple_animal/guardian/types/toy.dm [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+/mob/living/simple_animal/hostile/guardian/toy/verb/Battlecry()
+	set name = "Set Battlecry"
+	set category = "Guardian"
+	set desc = "Choose what you shout as you limply slap people."
+	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
+	if(input)
+		if(CHAT_FILTER_CHECK(input))
+			to_chat(src, "<span class='warning'>Your battlecry may not include prohibited words! Consider rereading the server rules.</span>")
+        	return
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/guardian/types/toy.dm]: Same here, tabs were converted to spaces.

**Comment by zeskorion** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Pull request merged**